cmd/cmd.go

```diff
@@ -804,19 +804,17 @@ func generate(cmd *cobra.Command, opts runOptions) error {
 }
 
 func RunServer(cmd *cobra.Command, _ []string) error {
-	host, port, err := net.SplitHostPort(strings.Trim(os.Getenv("OLLAMA_HOST"), "\"'"))
+	// retrieve the OLLAMA_HOST environment variable
+	client, err := api.ClientFromEnvironment()
```

Comment:
Feels a bit weird to be calling `ClientFromEnvironment` here

Reply:
Yep, hence the comment.

Reply:
probably should just add a utility function to cmd that returns the normalized host:port